KappaAnalysis/interface/KappaEventProviderBase.h

```diff
@@ -36,7 +36,8 @@ class KappaEventProviderBase: public EventProviderBase<TTypes> {
 		m_fi.SpeedupTree(128*1024*1024); // in units of bytes
 
 		// auto-delete objects when moving to a new object. Not default root behaviour
-		m_fi.eventdata.SetAutoDelete(true);
```

Comment:
Can you help us finding the commit you are talking about? Do you have the github link?